src/admin/helpers/MenuHelper.php

```diff
@@ -253,8 +259,9 @@ public static function getWebsites()
             self::$websites = (new Query())
                 ->select(['cms_website.id', 'cms_website.name', 'cms_website.host', 'cms_website.is_default', 'default_container_id' => 'cms_nav_container.id'])
                 ->from('cms_website')
-                ->leftJoin('cms_nav_container', 'website_id = cms_website.id')
+                ->innerJoin('cms_nav_container', 'website_id = cms_website.id')
                 ->where(['cms_website.is_active' => true, 'cms_website.is_deleted' => false])
+                ->groupBy('cms_website.id')
```

Comment:
message: "SQLSTATE[42000]: Syntax error or access violation: 1055 Expression #5 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'luya_env_dev.cms_nav_container.id' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by\nThe SQL being executed was: SELECT `cms_website`.`id`, `cms_website`.`name`, `cms_website`.`host`, `cms_website`.`is_default`, `cms_nav_container`.`id` AS `default_container_id` FROM `cms_website` INNER JOIN `cms_nav_container` ON website_id = cms_website.id WHERE (`cms_website`.`is_active`=TRUE) AND (`cms_website`.`is_deleted`=FALSE) GROUP BY `cms_website`.`id`"

Reply:
This will throw an exception,

Reply:
Fixed. I did use mariadb locally where the sql_mode is different.

Reply:
its not possible to use distinct? does any_value work with other systems like pgsql?

Reply:
Changed it to MIN aggregate func, will get the same result.